src/ScriptBuilder.Tests/OutboxScriptBuilderTest.cs

```diff
@@ -48,9 +45,7 @@ public void BuildDropScript(BuildSqlDialect sqlDialect)
         {
             SqlValidator.Validate(script);
         }
-        using (ApprovalResults.ForScenario(sqlDialect))
-        {
-            TestApprover.Verify(script);
-        }
+
+        Approver.Verify(script, scenario: "ForScenario." + sqlDialect);
```

Comment:
If we want we could (later) remove `"ForScenario." +` and rename the file, but this gets it to match the approval file that currently exists.